module/x/gravity/abci.go

```diff
@@ -364,9 +365,10 @@ func outgoingTxSlashing(ctx sdk.Context, k keeper.Keeper) {
 
 		// Slash unbonding validators who didn't sign new signer set txs only
 		if sstx, ok := otx.(*types.SignerSetTx); ok {
+			sstxHeight := int64(sstx.Height)
```

Comment:
Is this the same as `otxHeight`? Is one being retrieved through a function and one being retrieved directly? If it's the same you can just use the existing value.